### PR TITLE
Flutter 3.7.0

### PR DESCRIPTION
**Pull request by domesticmouse**
Changes required for `beta` to pass CI

## Pre-launch Checklist

- [x] I read the [Effective Dart: Style] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Effective Dart: Style]: https://dart.dev/guides/language/effective-dart/style
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: adaptive_app/codelab_rebuild.yaml [hunk 1]
```diff
@@ -1069,13 +1069,16 @@ steps:
...
-          < # homepage: https://www.example.com
+        patch-u: |
+          --- b/adaptive_app/step_06/yt_cors_proxy/pubspec.yaml
+          +++ a/adaptive_app/step_06/yt_cors_proxy/pubspec.yaml
+          @@ -1,5 +1,5 @@
+           name: yt_cors_proxy
+          -description: A server app using the shelf package and Docker.
+          +description: A YouTube CORS Proxy Server.
+           version: 1.0.0
+           # repository: https://github.com/my_org/my_repo
```
Comment (miquelbeltran) [inline adaptive_app/codelab_rebuild.yaml:1080]: The `# repository:...` commented out line, could probably be removed?

File: firebase-get-to-know-flutter/codelab_rebuild.yaml [hunk 2]
```diff
@@ -39,11 +40,13 @@ steps:
...
           +++ a/firebase-get-to-know-flutter/step_02/android/app/build.gradle
-          @@ -47,7 +47,7 @@ android {
+          @@ -46,8 +46,8 @@ android {
+                   // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
                    applicationId "com.example.gtk_flutter"
                    // You can update the following values to match your application needs.
-                   // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
+          -        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
           -        minSdkVersion flutter.minSdkVersion
+          +        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
```
Comment (miquelbeltran) [inline firebase-get-to-know-flutter/codelab_rebuild.yaml:49]: Here only the line `minSdkVersion...` should be modified, and the rest should remain the same

**Review by miquelbeltran — APPROVED** [review]
LGTM! I have added a couple of comments for things that could be improved.

File: adaptive_app/codelab_rebuild.yaml (hunk 1 above)
Comment (domesticmouse) [reply adaptive_app/codelab_rebuild.yaml]: Done

**Pull request merged**